src/Common/perf/PerfRunner/PerfRunner.runtimeconfig.json

```diff
@@ -1,8 +0,0 @@
-{
-	"runtimeOptions": {
```

Comment:
Do we not need these runtimeconfig files? Do we generate these as well?

Reply:
They are now delivered as supplemental data here: https://github.com/dotnet/buildtools/pull/2168/files#diff-425cb0dcf83ca9680e4a736c021dabe8R121

Reply:
Yeah that is fine